WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -335,6 +337,66 @@ extension PushNotificationsManager {
 
         return true
     }
+
+    /// Handles a Quick Start Local Notification
+    ///
+    /// - Note: This should actually be *private*. BUT: for unit testing purposes (within ObjC code, because of OCMock),
+    ///         we'll temporarily keep it as public. Sorry.
+    ///
+    /// - Parameters:
+    ///     - userInfo: The Notification's Payload
+    ///     - completionHandler: A callback, to be executed on completion
+    ///
+    /// - Returns: True when handled. False otherwise
+    @objc func handleQuickStartLocalNotification(_ userInfo: NSDictionary, completionHandler: ((UIBackgroundFetchResult) -> Void)?) -> Bool {
+        guard let type = userInfo.string(forKey: Notification.typeKey),
+            type == Notification.local else {
+                return false
+        }
+
+        if WPTabBarController.sharedInstance()?.presentedViewController != nil {
+            WPTabBarController.sharedInstance()?.dismiss(animated: false)
+        }
+        WPTabBarController.sharedInstance()?.showMySitesTab()
+
+        completionHandler?(.newData)
+
+        return true
+    }
+}
+
+extension PushNotificationsManager {
+    func postNotification(for tour: QuickStartTour) {
+        deletePendingLocalNotifications()
+
+        let content = UNMutableNotificationContent()
+        content.title = tour.title
+        content.body = tour.description
+        content.sound = UNNotificationSound.default
+        content.userInfo = [Notification.typeKey: Notification.local]
+
+        guard let futureDate = Calendar.current.date(byAdding: .day,
+                                                     value: Constants.localNotificationIntervalInDays,
+                                                     to: Date()) else {
+            return
+        }
+        let triggerDate = Calendar.current.dateComponents([.year, .month, .day, .hour, .minute, .second],
+                                                          from: futureDate)
+        let trigger = UNCalendarNotificationTrigger(dateMatching: triggerDate, repeats: false)
```

Comment:
Do we need `triggerDate`? Can we just pass `futureDate` straight in, now?

Reply:
Ignore me. I see `UNCalendarNotificationTrigger` takes date components, not a date 🤦‍♂️

Reply:
No problem! Let me know if it's good to :shipit: